File: app/src/main/java/es/etologic/mahjongscoring2/injection/MahjongScoringApp.java
package es.etologic.mahjongscoring2.injection;

import dagger.android.AndroidInjector;
import dagger.android.DaggerApplication;

public class MahjongScoringApp extends DaggerApplication {

    @Override protected AndroidInjector<? extends MahjongScoringApp> applicationInjector() {
        return DaggerAppComponent.builder().create(this);
    }
}

File: app/src/main/java/es/etologic/mahjongscoring2/injection/MahjongScoringApp.kt
package es.etologic.mahjongscoring2.injection

import dagger.android.AndroidInjector
import dagger.android.support.DaggerApplication

class MahjongScoringApp : DaggerApplication() {
    
    override fun applicationInjector(): AndroidInjector<out MahjongScoringApp> {
        return DaggerAppComponent.builder().create(this)
    }
}
